dali/operators/reader/loader/video/video_loader_decoder_gpu.cc

```diff
@@ -56,16 +52,13 @@ cudaStream_t VideoLoaderDecoderGpu::GetCudaStream() {
     CUDA_CALL(cuDriverGetVersion(&driver_cuda_version));
     if (driver_cuda_version >= 11030 && driver_cuda_version < 11040) {
       DALI_WARN_ONCE("Warning: Decoding on a default stream. Performance may be affected.");
-      return 0;
+      return;
```

Comment:
Consider adding a comment saying that the default stream lease results in using the default cuda stream

Reply:
I think that the warning should say enough about what's happening here. I don't want to leave such a comment because this behavior is far from ideal and likely not final.